qa/rpc-tests/wallet_shieldcoinbase_ua_nu5.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+from wallet_shieldcoinbase import WalletShieldCoinbaseTest
+from test_framework.util import assert_equal
+
+class WalletShieldCoinbaseUANU5(WalletShieldCoinbaseTest):
+    def __init__(self):
+        super(WalletShieldCoinbaseUANU5, self).__init__()
+        self.account = None
+        # activate after initial setup, before the first z_shieldcoinbase RPC
+        self.nu5_activation = 109
+
+    def test_init_zaddr(self, node):
+        # this function may be called no more than once
+        assert(self.account is None)
+        self.account = node.z_getnewaccount()['account']
+        self.addr = node.z_getaddressforaccount(self.account)['unifiedaddress']
+        return self.addr
+
+    def test_check_balance_zaddr(self, node, expected):
+        balances = node.z_getbalanceforaccount(self.account)
+        assert_equal(balances['pools']['transparent'], 0)
+        assert_equal(balances['pools']['sprout'], 0)
+        assert_equal(balances['pools']['sapling'], 0)
```

Comment:
I don't understand this; it's still possible to shield to Sapling coinbase after NU5 activation (and Orchard shielding isn't yet implemented.)

Reply:
The `_nu5.py` tests are currently disabled, and won't be enabled until after support for NU5 is added to `z_getaddressforaccount` and `z_getbalanceforaccount`. At that time, the unified address that this test uses as the shielding destination (generated using `z_getaddressbyaccount` with default arguments) will contain Orchard and Sapling receivers as those will be the "two best" (plus a `P2PKH` receiver), and then `z_shieldtocoinbase` will send to the best available pool, Orchard, and then this check will be correct.

But this brings up something missing from this test. With NU5 activated, the test should verify that the shielding-destination UA can contain a Sapling address but no Orchard address (shielding should send to the Sapling receiver). Should the test be enhanced to do that?